README.md

```diff
@@ -46,27 +68,31 @@ modules:
       units: metric
 ```
 
-The module configuration can contain secrets from the secrets yaml prefixed with `.Secrets`
+The module configuration can contain secrets from the secrets YAML prefixed with `.Secrets`
 as shown in the example above. 
 
 ## Modules
 
+You can discover modules on Github using [Github Search](https://github.com/search?q=topic%3Alooking-glass+topic%3Amodule+language%3AGo&ref=simplesearch).
+
 ### Package Naming
 
 If your module uses a hyphen, which is not supported by Go, it will be assumed that the package name is the
 last path of the hyphenated name (e.g. `looking-glass` would result in a package name `glass`). If this is not
-the case for your module, the `Package` should be set in the module cofiguration to the correct package name and
+the case for your module, the `Package` should be set in the module configuration to the correct package name and
 should be documented in your module.
 
 ### Development
 
+To make your module discoverable on Github, add the topics `looking-glass` and `module`.
+
 Modules are parsed in [yaegi](http://github.com/traefik/yaegi) and must expose two functions to be loaded:
 
 #### NewConfig
 
 `NewConfig` exposes your configuration structure to looking glass. The function must return
 a single structure with default values set. The yaml configuration will be decoded into
-the returned structure so it should contain `yaml` tags for configuration to be decoded
+the returned structure, so it should contain `yaml` tags for the configuration to be decoded
```

Comment:
I'd suggest showing an example or linking to something that describes what struct tags are in Go, since your repo might attract people that do not know about the specifics of the language and this sentence might leave them puzzled IMO